src/screens/FeedScreen/Content.js

```diff
@@ -19,12 +20,11 @@ import {
   POST_TYPE_POLL,
   POST_TYPE_STANDARD
 } from '../../utils/constants';
-import {fonts, normalizeFontSizeByWidth} from '../../utils/fonts';
+import {fonts, normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {getCaptionWithTopicStyle} from '../../utils/string/StringUtils';
 import {getCommentLength} from '../../utils/getstream';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `normalize` from '../../utils/fonts' has been removed. Make sure that this function is not used anywhere else in the file or it will cause a reference error. Also, the import of `getCommentLength` from '../../utils/getstream' has been added. Ensure that this function is correctly exported from the mentioned path.
```diff
- import {fonts, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {fonts, normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {getCommentLength} from '../../utils/getstream';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->